### PR TITLE
removed trailing whitespace

### PR DESCRIPTION
**Pull request by stefan-hoeck**
This removed all trailing whitespace and newline characters from all .idr files by running [fix_whitespace -f](https://github.com/stefan-hoeck/idris-fix-whitespace) from the root directory (see also #935).

However, there are many more files with whitespace issues in the repo (see attached log file). If this is desired, I could fix (a subset of) those as well.

[List of other files](https://github.com/idris-lang/Idris2/files/5837427/log.txt).

### CODE REVIEW

File: src/Utils/Path.idr [hunk 1]
```diff
@@ -1,541 +1,1081 @@
...
-export
-dropExtension : String -> String
-dropExtension path = path <.> ""
+module Utils.Path
+
+
+
+import Data.List
+
+import Data.Maybe
```
Comment (stepancheg) [inline src/Utils/Path.idr:7]: This change seems to be accidental.

Comment (stefan-hoeck) [reply src/Utils/Path.idr:7]: Oh dear, haha. I'll look into it.

Comment (stefan-hoeck) [reply src/Utils/Path.idr:4]: Seems like src/Utils/Path.idr contained Windows-style CRLF linebreak sequences and fix_whitespace was not yet able to handle those correctly. Both issues should now be fixed.

**Comment by stefan-hoeck** [comment]
About the failing Nix test: I'm also a Nix user, but I'm afraid I have no idea what's going on there.

**Comment by stepancheg** [comment]
Nix build fails on may PRs, it's unrelated to your changes.

**Pull request merged**